realm/realm-library/src/androidTest/java/io/realm/UTFStrings.java

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Maybe append `Test` to the file name to make it stand our that it contains tests.

Reply:
Yup, I renamed the class